scripts/devenv-builder/configure-vm.sh

```diff
@@ -35,16 +41,18 @@ if [ "$(whoami)" != "microshift" ] ; then
 fi
 
 # Check the subscription status and register if necessary
-if ! sudo subscription-manager status >& /dev/null ; then
-   sudo subscription-manager register
+if $RHEL_SUBSCRIPTION ; then
```

Comment:
Better to test explicity (`[ "$RHEL_SUBSCRIPTION" = true ]`), because as written the expression would evaluate to true even if the var is unset or empty.